boskos/README.md

```diff
@@ -3,3 +3,11 @@
 Boskos has been moved into its own repository under kubernetes-sigs:
 
 https://github.com/kubernetes-sigs/boskos
+
+# What's still here
+
+The only thing matters here is `./cmd/janitor/gcp_janitor.py` script, which is
+still used by
+[/scenarios/kubernetes_janitor.py#L66](https://github.com/kubernetes/test-infra/blob/d641897cea52d493ef883a4dfa6728ffdfa02dfa/scenarios/kubernetes_janitor.py#L66),
```

Comment:
I forget, is boskos using a different copy of this when deployed to janitor GCP or is this also the main janitor script?

Reply:
boskos has it's own `gcp_janitor.py` script https://github.com/kubernetes-sigs/boskos/blob/master/cmd/janitor/gcp_janitor.py, which is much more recent this this one, assuming the other one is the default?

Reply:
I don't know 🙈 , Jeff was probably the last person to know.

Reply:
Just in case @ixdy still receives GitHub notification ^^

Reply:
I *think* that the Boskos janitor uses the script from `kubernetes-sigs/boskos`, since I think it uses the copy baked into the container, which would come from that repo.

I opened #17719 regarding the copy of `gcp_janitor.py` in test-infra. There was a potentially hairy cleanup effort, and it also wasn't clear whether the jobs even needed to run anymore. With the resolution of #20760, this can probably finally be deleted.

Reply:
tempted to just disable scheduling those and see what breaks :+)